common-app-lib/src/cmr/common_app/services/kms_lookup.clj

```diff
@@ -28,8 +28,7 @@
    [cmr.common.util :as util]
    [cmr.redis-utils.config :as redis-config]
    [cmr.redis-utils.redis-hash-cache :as rhcache]
-   [cmr.transmit.kms :as kms])
-  (:import (clojure.lang ExceptionInfo)))
```

Comment:
Are you sure you don't need the import for the exception? Usually these have been put into the namespaces to avoid the class not found exception. when the compiler removes the ExceptionInfo since it thinks that it isn't being used.

Reply:
Resolving because this was undone but git is not showing as Outdated